INSTALL.md

```diff
@@ -147,7 +157,15 @@ you'll still need to do that yourself.
 
 # Notes for Windows Users
 
-Windows likes to make things difficult.  Common and historical issues:
```

Comment:
Not sure why you removed this line; especially...

Reply:
Because it discourages people. I have no strong opinion. It can also stay.

Reply:
Added it back.